packages/gibsonify_api/lib/src/models/login_info.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:equatable/equatable.dart';
+import 'package:uuid/uuid.dart';
+
+class LoginInfo extends Equatable {
+  LoginInfo({this.employeeName, this.employeeId, String? id})
+      : id = id ?? const Uuid().v4();
```

Comment:
Where is the uuid of a LoginInfo used? Do we need it?

Reply:
Needed to introduce this since the textformfields weren't rebuilding on app start. The state would be successfully loaded from storage but the fields themselves did not update. Needed to introduce keys for this to work (`UniqueKey` also didnt work) hence the ids.

Reply:
Hmmm, seems weird that `UniqueKey` does not work, I'll add a `TODO:` to look into this in the future, as this seems like information that we shouldn't need to store in the api for widgets to work correctly.